urdf_usd_converter/_impl/convert.py

```diff
@@ -62,6 +64,19 @@ def convert(self, input_file: str, output_dir: str) -> Sdf.AssetPath:
         parser = URDFParser(input_path)
         parser.parse()
 
+        # Get the package name and path of the ROS package from the CLI arguments
+        ros_packages = {}
+        for package in self.params.ros_packages:
+            if package.get("name", None) and package.get("path", None):
+                ros_packages[package.get("name")] = package.get("path")
+
+        # Search for ROS packages that reference meshes and material textures within URDF files.
```

Comment:
I forgot, we still need to update the mapping document with the new approach (reversing the previously suggested order of precedence)